stage #1/module #2. Strings/lesson #1. String/content.md

```diff
@@ -1,32 +1,38 @@
 # String (java.lang.String)
 
 ## Overview
-The String class represents character strings.
+The String class represents sequences of characters.
```

Comment:
As I can see, "character strings" is correct. You can see it [here](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html)